solver/llbsolver/ops/exec.go

```diff
@@ -563,13 +563,13 @@ func (e *ExecOp) sendLocally(ctx context.Context, root executor.Mount, mounts []
 	dstIndex := len(meta.Args) - 1
 	numFiles := dstIndex - i
 	if numFiles <= 0 {
-		return fmt.Errorf("SendFileMagicStr args missing; should be SendFileMagicStr [--dir] [--] <src> [<src> ...] <dst>")
+		return errors.Errorf("SendFileMagicStr args missing; should be SendFileMagicStr [--dir] [--] <src> [<src> ...] <dst>")
 	}
 	files := meta.Args[i:dstIndex]
 	dst := meta.Args[dstIndex]
 
 	if len(mounts) != 1 {
-		return fmt.Errorf("SendFileMagicStr must be given a mount with the artifacts to copy from")
+		return errors.Errorf("SendFileMagicStr must be given a mount with the artifacts to copy from")
```

Comment:
Any reason to switch from the standard `fmt` library to the deprecated (archived) `github.org/pkg/errors`? Is lint barking?

I am keen to reduce the number of dependencies after the release.  E.g. replace `github.com/hashicorp/go-multierror` and `github.com/pkg/errors` with standard `errors` and `fmt.Errorf()` libraries – a good and easy chore for somebody from the community to complete.

Reply:
Lint is indeed barking that we should not be using `fmt.Errorf`. We can somehow disable that lint if you think that's not a good change? (Is the problem that the lint / go version is out of date and it's recommending old things?)

Reply:
Please ignore my comment!

My bad. I forgot it's not our code but a fork from https://github.com/moby/buildkit. They still use pkg/errors and hashicorp/go-multierror.